From 5bff2aa2ad0683cbfcdd3198bd2113c7ecb4f725 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Sat, 10 Mar 2012 00:36:47 +0800
Subject: ENGR00176812-11 mx6 sabreauto: remove section mismatch warning

Remove the followinig section mismatch warning
WARNING: vmlinux.o(.text+0x1bdf8): Section mismatch in
reference from the function gpmi_nand_platform_init() to
the (unknown reference) .init.data:(unknown)
The function gpmi_nand_platform_init() references
the (unknown reference) __initdata (unknown).
This is often because gpmi_nand_platform_init lacks a __initdata
annotation or the annotation of (unknown) is wrong.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index fe7431f..63bd843 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -259,7 +259,7 @@ static const struct esdhc_platform_data mx6q_sabreauto_sd1_data __initconst = {
 };
 
 
-static int gpmi_nand_platform_init(void)
+static int __init gpmi_nand_platform_init(void)
 {
 	iomux_v3_cfg_t *nand_pads = NULL;
 	u32 nand_pads_cnt;
-- 
1.7.9.5

